.github/workflows/pre-commit.yml

```diff
@@ -36,6 +33,34 @@ jobs:
         run: |
           git config --global --add safe.directory "${GITHUB_WORKSPACE}"
           git config --global --list
+      - name: Install build dependencies
+        run: tools/debian_buildenv.sh setup
```

Comment:
This is a quite costly operation IMO - how do you feel about updating the Docker image instead? Note that would could also make the container in #14018 pushed in GHCR and reuse it in our CI? It may actually be quite good as we could use it `build` (Ubuntu) as well and a save a significant amount of bandwidth/resource consumption

Reply:
yeah, I agree. this was just a quick and dirty PoC that has stalled now (because of issues like these). Updating the docker would def. be preferred.

Reply:
@mixxxdj/developers any objection in me extending the scope of https://github.com/mixxxdj/mixxx/pull/14018 and publish devcontainer in Mixxx GHCR repo?